src/pages/home/report/ReportDropUI.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {View} from 'react-native';
+import withLocalize, {withLocalizePropTypes} from '../../../components/withLocalize';
+import DragAndDropIcon from '../../../../assets/images/drag-and-drop.svg';
+import styles from '../../../styles/styles';
+import Text from '../../../components/Text';
+import CONST from '../../../CONST';
+import DropZone from '../../../components/DragAndDrop/DropZone';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+};
+
+const ReportDropUI = props => (
+    <DropZone hostName="ReportDropZone" dropZoneId={CONST.REPORT.ACTIVE_DROP_NATIVE_ID}>
```

Comment:
Extract this portal hostname to a constant

Reply:
+1